### ISSUE TITLE
a11y for Message Box

### ISSUE THREAD

**Issue by SAPAndiBraimllari** [SAP/fundamental-ngx#12180]
#### Is this a bug, enhancement, or feature request?

Bug

#### Describe your proposal.

**_"This element (role=dialog) does not have a mechanism that allows an accessible name value to be calculated"_**

A11y issues regarding the Message Box. Basically, the `role=dialog` and no `aria-label` can be assigned to it
I guess we either remove https://github.com/SAP/fundamental-ngx/blob/main/libs/core/message-box/message-box.component.ts#L38 if it isn't strictly required, or maybe allow the user to add an `aria-label`. 

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

Fundamental version of ^0.50.0

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

Create a message box by following https://fundamental-ngx.netlify.app/#/core/message-box#object-based

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.

We ran the AMP tool.

#### Is there anything else we should know?

We simply want to improve a11y for a message box.

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by InnaAtanasova** [comment]
Hi @SAPAndiBraimllari , thank you for opening the issue. 
If you have time please take a look at [this PR](https://github.com/SAP/fundamental-ngx/pull/12185). Thank you
